lib/ads/media_tailor_ad.js

```diff
@@ -22,6 +24,8 @@ shaka.ads.MediaTailorAd = class {
     /** @private {?mediaTailor.Ad} */
     this.ad_ = mediaTailorAd;
 
+    this.skipOffset_ = this.parseTime_(this.ad_.skipOffset);
```

Comment:
This should have JSDoc, so it'll be marked as `@private`.

Reply:
Done!